core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/SnapshotApiService.scala

```diff
@@ -63,25 +63,6 @@ trait SnapshotApiService extends UserInfoDirectives {
           snapshotServiceConstructor(userInfo).getSnapshotByName(WorkspaceName(workspaceNamespace, workspaceName), referenceName)
         }
       }
-    } ~
-    path("workspaces" / Segment / Segment / "snapshots" / "v2" / Segment) { (workspaceNamespace, workspaceName, snapshotId) =>
-      get {
-        complete {
-          snapshotServiceConstructor(userInfo).getSnapshot(WorkspaceName(workspaceNamespace, workspaceName), snapshotId)
-        }
-      } ~
-      patch {
-        entity(as[UpdateDataReferenceRequestBody]) { updateDataReferenceRequestBody =>
-          complete {
-            snapshotServiceConstructor(userInfo).updateSnapshot(WorkspaceName(workspaceNamespace, workspaceName), snapshotId, updateDataReferenceRequestBody).map(_ => StatusCodes.NoContent)
-          }
-        }
-      } ~
-      delete {
-        complete {
-          snapshotServiceConstructor(userInfo).deleteSnapshot(WorkspaceName(workspaceNamespace, workspaceName), snapshotId).map(_ => StatusCodes.NoContent)
-        }
-      }
```

Comment:
apparently we defined these twice? As a copy/paste error? This block is exactly the same as lines 41-59, so I removed it.